visualizer/README.md

```diff
@@ -1,15 +1,22 @@
 # MIPT-V Module topology visualizer tool
+Application fetches the topology data generated by the special MIPT-V command line option and visualizes it in an interactive graph

```

Comment:
That does not really explain anything. What is the motivation for that tool? What is "topology"?